src/components/common/Select/index.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import SelectReact from 'react-select';
+
+const activeColor = '#1f1f1f';
+const backgroundColor = '#538d4e';
+const color = 'white';
+const focusedColor = '#1f1f1f';
+const hoverColor = '#c6d4c5';
+const selectedColor = '#3d6b39';
+
+const Select = ({ ...props }) => {
+  const colorStyles = {
```

Comment:
We can move this object out of the component. I can do that when I commit the thing about the blue border